test/alternator/test_batch.py

```
@@ -614,6 +616,77 @@ def test_batch_get_item_partial(scylla_only, dynamodb, rest_api, test_table_sn):
+    items = [{'p': p + str(i), 'c': i} for i in range(2)]
+    with test_table_sn.batch_writer() as batch:
+        for item in items:
+            batch.put_item(Item=item)
+
+    request_items = {test_table_sn.name: {'Keys': items, 'ConsistentRead': True}}
+    pause_injection = "alternator_batch_get_item_pause_later_read"
+    failure_injection = "alternator_batch_get_item_response_failure"
+    injection_parameters = {'table_name': test_table_sn.name}
+
+    def injection_enter_count(injection):
+        response = requests.get(f'{rest_api}/v2/error_injection/injection/{injection}/enters')
```

Ruff (0.16.2)

[error] Probable use of `requests` call without timeout

(S113)